pages/review.js

```diff
@@ -391,7 +391,11 @@ export default function Review(props) {
           </>
         )}
         {!(counter < questions.length) && (
-          <OpenPR answer={result} projectName={props.submission.name} />
+          <OpenPR
+            answer={result}
+            projectName={props.submission.name}
+            startTime={props.start_time}
```

Comment:
If you change the one above, then again here you would use `startTime={props.startTime}`. Given that all the codebase uses camelCase, I see no reason to break the consistency here :)

Reply:
@lacabra I try to avoid using the same name for the parameter and the state because I tend to get confused about what is being referenced and sometimes it has led to errors as well. But I agree that here it is better to keep the names consistent. I have made the required changes in commit a657fd5.